services/seleniumService.py

```diff
@@ -43,6 +43,25 @@ def insertUserStatus(driver,currentStatus):
     dropDownRoot=menu.find_element(By.CLASS_NAME,"MuiPaper-root").find_element(By.CLASS_NAME,"MuiList-root")
     dropDownRoot.find_element(By.XPATH,"/html/body/div[3]/div[3]/ul/li[contains(text(),'"+currentStatus+"')]").click()
    
+def insertExpirationDate(driver,currentStatus_Expires):
+    driver.find_element(By.ID,"Expiration-Date").send_keys(currentStatus_Expires)
+
+def insertCompanyName(driver,petitioner):
+    companys=driver.find_element(By.ID,"sponsor-company-autocomplete")
+    searchCompany=companys.find_element(By.CLASS_NAME,"MuiAutocomplete-input")
+    searchCompany.send_keys(petitioner)
+
+def selectCompany(driver):
+    ActionChains(driver).key_down(Keys.ARROW_DOWN).key_up(Keys.ARROW_DOWN).perform()
+    ActionChains(driver).key_down(Keys.ENTER).key_up(Keys.ENTER).perform()
+
+def insertEmail(driver,userEmail):
+    email=driver.find_element(By.XPATH,"/html/body/div[2]/section/div[13]/div[2]/div")
+    email.click()
+    driver.find_element(By.XPATH,"/html/body/div[2]/section/div[13]/div[2]/div/div/input").send_keys(userEmail)
+
+def clickSaveProfile(driver):
+    driver.find_element(By.ID,"profile-modal-save").click()
```

Comment:
This is a small nitpick, but it's best practice to try and maintain a similar structure throughout your code, as it'll make things easier to follow and read. In this specific case, the functions you wrote have varying structure.

See how on line 59-60, we are splitting up the finding of the element and the clicking of the element, yet in line 64 we do that all in one line. It would be better to try and follow a similar pattern between your functions.

I personally prefer the former approach, as splitting things up into multiple lines is often much easier to debug and read. Whatever you prefer though, it would be best to follow that pattern whenever and wherever you can.

This should not effect the end result of your code, though it will help you write better/more consistent code, and its just generally a good practice to follow.

```suggestion
    saveButton=driver.find_element(By.ID,"profile-modal-save")
    saveButton.click()
```

Reply:
I was actually originally trying to to that inset email function as one line but it wouldn't find the right element. 

For some reason it worked after breaking it up like that. But it also could have been I was doing something wrong that I fixed as I cleaned it up.